db/seeds.rb

```diff
@@ -5,7 +5,67 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
-first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.')
-first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')
+# Users
```

Comment:
- [ Required ] Good job adding this seed file to your project, however when I run `rails db:seed` it raises an error `ActiveModel::UnknownAttributeError: unknown attribute 'author' for Comment` as shown below, I suggest fixing the error or emptying the contents of the file.

## Project requirements
![Screenshot from 2023-02-11 01-53-10](https://user-images.githubusercontent.com/16392046/218217811-a9f323b2-4dab-4c92-b27d-cbd58463c7c9.png)